### ISSUE TITLE
⌘ as Meta on Macs

### ISSUE THREAD

**Issue by pasha-bolokhov** [tuttieee/vscode-emacs-mcx#221]
Could you add an option for the Command key (⌘) to work as Meta on Macs? Or, the "Option" key as Meta. You could give users a choice in the settings

I always use Meta, and when you hit ⌘-M, instead of "yanking" the selected text, it just closes the entire tab...
On Linux, usually Alt is treated as Meta, but on Macs I think the Command Key is probably the best, although, there is also "Option" key

**Comment by flindeberg** [comment]
It is possible to add this to the package settings.

I actually did this earlier today in another emacs-plugin for vscode (https://github.com/flindeberg/Emacs/blob/master/package.json), it looks like this (example):
            
           {
                "key": "alt+shift+.",
                "mac": "cmd+shift+.",
                "command": "emacs.cursorBottom",
                "when": "editorTextFocus"
            },

Where "key" is the general keybinding, and "mac" specific for OSX ("win" and "linux" also exist). 

I'm currently testing emacs-like-plugins for vscode, and if I like this one I might drop a PR for this.

**Comment by flindeberg** [comment]
Added PR #244 as a fix for this. 

Default original emacs behavior is cmd as meta, although some later version (weirdly enough) use option for meta. #244 emulates below emacs config:  

```
    (setq mac-option-key-is-meta nil)
    (setq mac-command-key-is-meta t)
    (setq mac-command-modifier 'meta)
    (setq mac-option-modifier nil)
```

**Comment by pasha-bolokhov** [comment]
Should probably start this as a separate "issue" — Emacs has a whole bunch of key-bindings (probably a hundred), most of them are *very* useful. In particular, the keys for working with rectangular selections...
I should probably start this as a separate issue on "rectangular", since I also have other requests

**Comment by tuttieee** [comment]
I close this issue because it's resolved by #244, thanks to @flindeberg.
The feature has been released in v0.19.0.
Note that you have to set `useMetaPrefixMacCmd` config to enable the keybindings (see #254)

**Comment by tuttieee** [comment]
And please create separate issues for other kinds of feature requests, as you said.
Thank you.
